Makefile

```diff
@@ -5,26 +5,35 @@ deps:
 	mix deps.compile
 compile: deps
 	mix compile
+
+release: export MIX_ENV = prod
+release: deps
+	mix release
+
 run: export BOT_TOKEN = $(shell cat bot.token)
 run: export PRICES_PATH = path/to/file
 run: compile
 	mix run --no-halt
+
 clean:
 	rm -rf _build
+
 purge: clean
 	rm -rf deps
 	rm mix.lock
 
-# scompile stands for systemd compile
-scompile:
+sinstall:
 	mix local.hex --force
 	mix local.rebar --force
-	mix deps.get
-	mix compile
+
+# scompile stands for systemd compile
+scompile: sinstall
+scompile: deps
+scompile: release
```

Comment:
Can we change this to:
`scompile: sinstall deps release`
??

Reply:
Changed #IHaveNoIdeaOfMake